salt/modules/disk.py

```diff
@@ -24,7 +24,7 @@ def __virtual__():
     Only work on POSIX-like systems
     '''
     if salt.utils.is_windows():
-        return False
+        return (False,'This module doesn''t work on Windows.')
```

Comment:
The single quotes in the middle of this string will actually end the first string and start a second string. You need to escape the apostrophe:

```
        return (False,'This module doesn\'t work on Windows.')
```